src/components/books.js

```diff
@@ -0,0 +1,24 @@
+/* eslint-disable react/jsx-key */
+/* eslint-disable arrow-body-style */
```

Comment:
### 🛑 Required Change

- [ ] Kindly avoid disabling linters because they guide us in writing code that is clean and well formatted.